### PR TITLE
doc / rate oracle feature

### PR DESCRIPTION
**Pull request by JeremyKono**
Rate oracle feature docs

### CODE REVIEW

File: content/features/rate-oracle.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Rate Oracle
+description:
+---
+
+import Callout from "../../src/components/Callout";
+
+import Prompt from "../../src/components/Prompt";
+
+The feature rate oracle conversion is used to convert any fiat or to other token or coin that you want to display. It pulls data from sources then converts your token to the desired currency.
```
Comment (Nullably) [inline content/features/rate-oracle.mdx:10]: I'd put it more like.
This new feature provides real time, most up-to-date exchange rate on any given token or currency from a reliable and trustworthy data source.

File: content/features/rate-oracle.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+<Callout
+  type="note"
+  body="The implementation of the conversion only is used [cross exhange market making] and [arbitrage] strategy."
```
Comment (Nullably) [inline content/features/rate-oracle.mdx:14]: The oracle is used in these two strategies but that's not its limitation.

File: content/features/rate-oracle.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  link={["/strategies/cross-exchange-market-making/", "/strategies/arbitrage/"]}
+/>
+
+## `rate_oracle_source`
+
+The source in where you want to pull data from it can either be Binance or Coingecko. Take note that using Coingecko will have a 30 second delay due to their API rate limit.
```
Comment (Nullably) [inline content/features/rate-oracle.mdx:20]: Some grammar error here.

File: content/features/rate-oracle.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+** Prompt: **
+
+<Prompt
+  prompt="What source do you want rate oracle to pull data from? (binance, coingecko)"
+  response=">>> "
+/>
+
+## `global_token`
+
+Global token you want to be displayed and converted currency to show.
```
Comment (Nullably) [inline content/features/rate-oracle.mdx:31]: This is a token which you can display other tokens' value in. For example, you can nicely see all your tokens balances in USD (if that your global token choice).

File: content/features/rate-oracle.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+** Prompt: **
+
+<Prompt
+  prompt="What is your default display token? (e.g. USDT,USD,EUR)"
+  response=">>> "
+/>
+
+## `global_token_symbol`
+
+Choose the symbol of the global token selected.
```
Comment (Nullably) [inline content/features/rate-oracle.mdx:42]: A symbol for the global token.

File: content/features/rate-oracle.mdx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  response=">>> "
+/>
+
+## How it works
+
+For the rate oracle conversion to show after the `balance`, `pnl`, `open_orders`, `trades`, and `status` command, You have to edit in `config_global.yml` to change the `rate_oracle_source`, `global_token` and `global_token_symbol` or you can edit it by using the `config` command.
+
+Refer to the example below:
+
+Changed the default setting in `conf_global.yml` to GBP (Great Britain Pound). The conversion will show up when you run `balance` command.
```
Comment (Nullably) [inline content/features/rate-oracle.mdx:57]: Change, not changed.

**Comment by JeremyKono** [comment]
All requested changes addressed in the current commit

**Pull request merged**